Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainActivity.kt

```diff
@@ -88,6 +89,7 @@ class MainActivity : AppCompatActivity(), HasAndroidInjector {
         scheduleWork()
         checkShouldDisplayBackgroundWarning()
         doBackgroundNoiseCheck()
+        scheduleDeadmanNotification()
```

Comment:
We don't really need an extra method if it just calls another one line method.

Do we need to do this in `onResume` though?

Could we use something like `ForegroundState.kt` to just execute it any time the app is moved from background to foreground?

Reply:
Removed extra method.
Right now it would require additional checks if we want to place this call in ForegroundState (onboarding is finished).
I would say it should be streamlined with other background workers if we want to switch to ForegroundState, mb as part of BackgroundWorkScheduler refactoring in future.